domenetjenester/behandling/src/main/java/no/nav/foreldrepenger/behandling/impl/HistorikkInnslagForAksjonspunktEventObserver.java

```diff
@@ -43,12 +43,12 @@ public HistorikkInnslagForAksjonspunktEventObserver(Historikkinnslag2Repository
         var ktx = aksjonspunkterFunnetEvent.getKontekst();
         for (var aksjonspunkt : aksjonspunkterFunnetEvent.getAksjonspunkter()) {
             if (aksjonspunkt.erOpprettet() && AksjonspunktDefinisjon.AUTO_KØET_BEHANDLING.equals(aksjonspunkt.getAksjonspunktDefinisjon())) {
-                opprettHistorikkinnslagForVenteFristRelaterteInnslag(ktx.getBehandlingId(), ktx.getFagsakId(), "Behandlingen er satt på vent", null,
+                opprettHistorikkinnslagForVenteFristRelaterteInnslag(ktx.getBehandlingId(), ktx.getFagsakId(), "Behandling er satt på vent", null,
```

Comment:
ser at vi bruker både `Behandling er` og `Behandlingen er`. Kanskje vi bare skal gå for _Behandlingen_?